katas/content/superposition/hardy_state/Solution.qs

```diff
@@ -0,0 +1,14 @@
+namespace Kata {
+    open Microsoft.Quantum.Math;
+
+    operation Hardy_State (qs : Qubit[]) : Unit is Adj {
+        // Follow Mariia's answer at https://quantumcomputing.stackexchange.com/questions/6836/how-to-create-quantum-circuits-from-scratch
```

Comment:
```suggestion
```
Probably we can remove this from here now, since this link is already in `solution.md` file. In previous version of Katas, we didn't have Workbooks from the start,  people used to refer solutions from `ReferenceImplementation.qs`, so that's why we have detailed comments in some solutions.